spec/swagger.yaml

```diff
@@ -2967,36 +3030,46 @@ paths:
           in: path
           required: true
           type: string
-          description: ''
-        - name: request
-          description: ''
-          in: body
-          required: true
-          schema:
-            $ref: '#/definitions/WatcherRequest'
+          description: Unique identifier of corresponding account
+        - name: email
+          in: query
+          required: false
+          type: string
+          description: Unique identifier of corresponding account
+        - name: limit
+          in: query
+          required: false
+          default: 100
+          type: number
+          description: limit
```

Comment:
@jones-smartling @hchaps, seems like various apis use different description for `limit` and `offset` parameters. Do we want them to be consistent across all apis?

Reply:
Might need @hchaps to help standardize the language here, but for now its ok. We wont get consistent across defaults and such and that is ok. My 2 cents.